scripts/publish-image-runtime

```diff
@@ -5,11 +5,21 @@ cd $(dirname $0)/..
 
 source ./scripts/version.sh
 
-set +x
-docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-set -x
-
-docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-${GOOS}-${GOARCH}
-if [ "${GOARCH}" != "s390x" ] && [ "${GOARCH}" != "arm64" ]; then
-    docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64
-fi
+./scripts/setup-docker-builder
+
+DOCKER_BUILDKIT=${DOCKER_BUILDKIT:-1} docker image build ${IID_FILE_FLAG} \
```

Comment:
The publish-image action calls the make target, and sets IID_FILE_FLAG for use by cosign.

https://github.com/rancher/ecm-distro-tools/blob/master/actions/publish-image/action.yaml#L166-L174